Tools/sensibo.60s.sh

```diff
@@ -192,6 +222,13 @@ do
         POD_TEMPERATURE=$POD_TEMPERATURE_ORIGINAL
     fi
 
+    # If climate react enabled is set to true then relable as ON, else relable as OFF
+    if [ "$POD_CLIMATE_REACT_ENABLED" = "true" ]; then
+        POD_CLIMATE_REACT_STATUS=$(echo "ON")
```

Comment:
```
        POD_CLIMATE_REACT_STATUS=$(echo "ON")
                                 ^----------^ SC2116: Useless echo? Instead of 'cmd $(echo foo)', just use 'cmd foo'.
```